tsconfig-base.json

```diff
@@ -1,7 +1,8 @@
 {
   "compilerOptions": {
     "moduleResolution": "Node",
-    "target": "ESNext",
+    "target": "ES2022",
+    "lib": ["ESNext"],
```

Comment:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-5-2.html#using-declarations-and-explicit-resource-management

These are recommended at the bottom of the section.